torch/_ops.py

```diff
@@ -768,7 +776,14 @@ def __init__(self, name, ops):
         self._ops = ops
 
     def __getattr__(self, name):
-        return self._ops[name]
+        # Following _OpNamespace.__getattr__, we cache the op on the _PyOpNamespace object.
+        op = self._ops.get(name, None)
+        if op is None:
+            raise AttributeError(
+                f"'_PyOpNamespace' '{self.name}' object has no attribute '{name}'"
+            )
+        setattr(self, name, op)
+        return op
```

Comment:
Is this necessary for this PR? Or is it just an optimization?

Reply:
yeah, it's just an optimization. I was looking at how other opearator's \_\_module\_\_ is set and found this logic and the error message seems pretty good. So I "copied" it over.

Reply:
Sounds good, thanks for explaining